src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -137,15 +183,23 @@ class ActivityResourceIntTest : IntegrationTestBase() {
         within(BigDecimal("0.001"))
       )
       assertThat(courseActivity.payPerSession).isEqualTo(PayPerSession.F)
+      assertThat(courseActivity.courseSchedules.first().scheduleDate).isEqualTo("2022-10-31")
     }
 
     private fun callCreateEndpoint(): Long {
       val response = webTestClient.post().uri("/activities")
         .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_ACTIVITIES")))
         .contentType(MediaType.APPLICATION_JSON)
-        .body(
-          BodyInserters.fromValue(
-            """{
+        .body(BodyInserters.fromValue(validJsonRequest()))
+        .exchange()
+        .expectStatus().isCreated
+        .expectBody(CreateActivityResponse::class.java)
+        .returnResult().responseBody
+      assertThat(response?.courseActivityId).isGreaterThan(0)
+      return response!!.courseActivityId
+    }
+
+    private fun validJsonRequest() = """{
```

Comment:
Just moved the JSON to a function so I could reuse. Such ingenious refactoring is completely beyond the wit of git diff 😞